cloud/cloud_file_system.cc

```diff
@@ -31,6 +35,33 @@
 
 namespace ROCKSDB_NAMESPACE {
 
+#ifdef USE_AWS
+std::shared_ptr<void> useAWS() {
+    static std::mutex gMutex;
+    static std::weak_ptr<void> gInstance;
+
+    std::lock_guard lock(gMutex);
+
+    auto existing = gInstance.lock();
+    if (existing) {
+        return existing;
+    }
+
+    Aws::InitAPI(Aws::SDKOptions());
+
+    // No need to actually allocate or delete real memory
+    static bool dummy;
+
+    auto rv = std::shared_ptr<void>(&dummy, [](bool*) {
+        // Shut down AWS API.
+        Aws::ShutdownAPI(Aws::SDKOptions());
```

Comment:
There is a Shutdown method in AwsFileSystem:

https://github.com/rockset/rocksdb-cloud/blob/master/cloud/aws/aws_file_system.h#L63

We should either remove it, or create a InitAPI method and let the caller handle the Aws API lifecycle